server/utils/bookings/index.ts

```diff
@@ -175,6 +175,23 @@ export const bookingActions = (user: UserDetails, booking: Booking): Array<Ident
   return []
 }
 
+export const v2BookingActions = (booking: Booking): Array<IdentityBarMenu> => {
+  if (booking.status === 'awaiting-arrival')
+    return [
+      {
+        items: [
+          {
+            text: 'Withdraw placement',
+            classes: 'govuk-button--secondary',
+            href: applyPaths.applications.withdraw.new({ id: booking.applicationId }),
+          },
+        ],
+      },
+    ]
```

Comment:
Quite likely I'm getting confused, but as the v2BookingActions will only be available to users with the `FUTURE_MANAGER` role what's the reason to limit the actions? 

We will ultimately be offering (almost) all the current features:

- update departure dates
- log a departure
- ~~withdraw an arrived placement~~
- move a person to a new bed
- log an arrival
- log a non-arrival

Perhaps you anticipate that these features will be significantly different in v2 manage and so prefer to start from a blank slate and build up incrementally as we port enhanced features over?

Reply:
Yes, your suspicion is correct. I think we are best copying over these features piece by piece as and when they are needed. 
If we copy them over now there will be a lot of duplicated code which we will have to alter slightly to fit the new approach which we will then have to test and support.
It should be easy enough to add these features when the time comes but for now I don't think it's worth us spending time on them when they *could* change drastically in the future.